lib/process.py

```diff
@@ -44,9 +44,10 @@
 ALL_TYPES, and for arrays with any number of dimension specified
 in DIMENSIONS and each type listed in ALL_TYPES. PREFIX can be
 used to add a prefix to static functions defined in the template.
-'''
 
+'''
 
+from __future__ import print_function
```

Comment:
"Teacher, teacher, I know something ... otherwise process.py crashes when using python2 and an error message is printed" (*grin* I just fixed the same bug on my branch 😊 Looks like I did all my testing with python3 ... though ... process.py is invoked from make, not explicitly from me ... so why did it work in the past??)

Reply:
A gold star to Joerg (or house points, or a merit or whatever...) :-)